packages/react-dom/src/__tests__/ReactDOMServerPartialHydration-test.internal.js

```diff
@@ -1561,10 +1561,9 @@ describe('ReactDOMServerPartialHydration', () => {
     const b = container.getElementsByTagName('span')[1];
     expect(b.textContent).toBe('B');
 
-    const root = ReactDOM.createRoot(container, {hydrate: true});
-
+    const root = ReactDOM.hydrateRoot(container);
```

Comment:
I don't understand why this test fails after this change. Have any ideas @acdlite  ? I looked over `createRoot` vs `hydrateRoot` and they seem to do the same thing except that `hydrateRoot` includes an `updateContainer` call which even when I make optional (only call if `initialChildren` is passed in) the test still fails O.o. I can't seem to tell what the difference is.